components/insight/SRC/org/openmicroscopy/shoola/env/data/DataServicesFactory.java

```diff
@@ -586,6 +588,19 @@ public void connect(UserCredentials uc)
         	return;
         }
 
+        // TODO: Can be removed for >= 5.5.0 release
+        container.getRegistry().bind(LookupNames.SERVER_5_4_8_OR_LATER, VersionCompare.compare(version, "5.4.8") >= 0);
+        
+        IConfigPrx cs = omeroGateway.getGateway().getConfigService(new SecurityContext(exp.getGroupId()));
+        try {
+            String val = cs.getConfigValue("omero.pixeldata.max_plane_width");
+            container.getRegistry().bind(LookupNames.MAX_PLANE_WIDTH, val != null ? Integer.parseInt(val) : 3192);
+            val = cs.getConfigValue("omero.pixeldata.max_plane_height");
+            container.getRegistry().bind(LookupNames.MAX_PLANE_HEIGHT, val != null ? Integer.parseInt(val) : 3192);
```

Comment:
sorry for adding comment a bit late
the default value of 3192 should have been put in container.xml so that if the server value is not set
it can at least be added client side

Reply:
That's true. Will do that in a separate PR. Don't think it'll block 5.4.8, does it?

Reply:
the PR has been merged so it will have to be done in a separate PR
It is not a blocker